ignite/metrics/mean_average_precision.py

```diff
@@ -0,0 +1,89 @@
+__all__ = ["MeanAveragePrecision"]
+
+from typing import Callable, List, Optional, Sequence, Union
+
+import torch
+from torchvision.ops import box_iou
+
+from ignite.metrics.metric import Metric
+
+
+class MeanAveragePrecision(Metric):
+    def __init__(
+        self,
+        iou_thresholds: Optional[Union[List[float], torch.Tensor]] = None,
+        output_transform: Callable = lambda x: x,
+        device: Union[str, torch.device] = torch.device("cpu"),
+    ) -> None:
+        if iou_thresholds is None:
+            iou_thresholds = torch.range(0.5, 0.99, 0.05)
+        self.iou_thresholds = torch.tensor(iou_thresholds, device=device)
+        self.rec_thresholds = torch.range(0, 1, 0.01, device=device)
```

Comment:
Maybe, `torch.linspace(0, 1, 100)` instead ?

Reply:
It's `torch.linspace(0, 1, 101)`